data/pigui/modules/radar.lua

```diff
@@ -17,7 +17,7 @@ local icons = ui.theme.icons
 local MAX_RADAR_SIZE = 1000000000
 local MIN_RADAR_SIZE = 1000
 
-local shouldDisplay2DRadar = false
+local shouldDisplay2DRadar
```

Comment:
I would have left this with a safe default of `false`, since the later cases are `elseif` not just `else` you could in theory end up with it being `nil`... I think

Reply:
Ive changed the later ones to "if then else end", no more elseif, it defaults to old radar unless new is activated